funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -973,6 +973,10 @@ def start(self, poll_period=None):
                         )
                     for b_message in b_messages:
                         r = pickle.loads(b_message)
+                        try:
+                            r["times"].update({"interchange_result": time.time()})
+                        except KeyError:
+                            r["times"] = {"interchange_result": time.time()}
```

Comment:
```suggestion
                        if "times" not in r:
                            r["times"] = {}
                        r["times"]["interchange_result"] = time.time()
```
One line shorter, but more importantly, I think this is easier to follow.